README.md

```diff
@@ -25,26 +24,179 @@ The package could be installed via composer:
+use Yiisoft\Db\Sqlite\Connection as SqliteConnection;
+use Yiisoft\Log\Logger;
+use Yiisoft\Profiler\Profiler;
+use Yiisoft\Yii\Db\Migration\Helper\ConsoleHelper;
+use Yiisoft\Yii\Db\Migration\Service\MigrationService;
+
+
+/** 
+ * Config both console and web.
+ *
+ * Add to existing configuration.
+ */
+return [
+    ConnectionInterface::class => static function (ContainerInterface $container){
+        $aliases = $container->get(Aliases::class);
+        $cache = $container->get(CacheInterface::class);
+        $logger = $container->get(LoggerInterface::class);
+        $profiler = $container->get(Profiler::class);
+
+        return new SqliteConnection(
+            $cache,
+            $logger,
+            $profiler,
+            'sqlite:' . $aliases->get('@runtime/yiitest.sq3')
+        );
+    },
+
+    MigrationService::class => static function (ContainerInterface $container) {
+        $migrationService = new MigrationService(
+            $container->get(ConnectionInterface::class),
+            $container->get(ConsoleHelper::class)
+        );
+
+        /** Namespace store migrations */
+        $migrationService->createNamespace('App\\Migration');
+
+        /** Namaspaces find migration all packages */
+        $migrationService->updateNamespace(['App\\Migration']);
+
+        return $migrationService;
+    }
+];
+```
+
+Now the `MigrationService::class` uses the `View` of the application that is already registered in `providers-web.php`, we must move its registry to `provider.php`, so that it is defined in the console and web respectively.
+
+config/providers.php
+```php
+<?php
+
+declare(strict_types=1);
+
+use App\Provider\ThemeProvider;
+use App\Provider\WebViewProvider;
+
+/** 
+ * Config both console and web.
+ *
+ * Add to existing configuration.
+ */
+
+return [
+    'yiisoft/view/theme' => [
+        '__class' => ThemeProvider::class,
+        '__construct()' => [
+            $params['yiisoft/view']['theme']['pathMap'],
+            $params['yiisoft/view']['theme']['basePath'],
+            $params['yiisoft/view']['theme']['baseUrl'],
+        ],
+    ],
+    'yiisoft/view/webview' => [
+        '__class' => WebViewProvider::class,
+        '__construct()' => [
+            $params['yiisoft/view']['basePath'],
+            $params['yiisoft/view']['defaultParameters'],
+        ],
+    ],
+];
+```
+
+Execute `composer du` in console config its rebuild.
+
+Now we have the `yiisoft/yii-db-migration` package configured and it can be called in the console.
+
+View the list of available commands execute in console: `vendor/bin/yii list`
+
+```
+Available commands:
+  database/list    Lists all tables in the database.
+  generate/create  Generate migration template.
+  help             Displays help for a command
+  list             Lists commands
+  migrate/down     Downgrades the application by reverting old migrations.
+  migrate/history  Displays the migration history.
+  migrate/new      Displays the migration history.
```

Comment:
Description is a duplicate of `migrate/history`

Reply:
fixed.